plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -25,6 +25,39 @@ class WC_Helper_Admin {
 	 */
 	public static function load() {
 		add_filter( 'woocommerce_admin_shared_settings', array( __CLASS__, 'add_marketplace_settings' ) );
+		add_filter( 'rest_api_init', array( __CLASS__, 'register_rest_routes' ) );
```

Comment:
Not sure if this is the best file to register this endpoint. `plugins/woocommerce/includes/admin/class-wc-admin-addons.php` file is thousand lines long.

Reply:
I think it makes sense for now. It keeps our code separate from the older code. Also, I see `class-wc-admin-addons.php` more as a render file for the marketplace. I know it does a lot more, but it feels right in this case to keep all our code together.

Reply:
Agreed! We will want to deprecate `class-wc-admin-addons.php` in future, so this is good.

Reply:
Thank you folks for the confirmation!